### PR TITLE
Prep for v 0.16.0

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Update changelog and package versions for v 0.16.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,17 @@
...
+
+### New features
+
+- Heading component (`gcds-heading`)
+  - New `gcds-heading` to render headings in GCDS styles
+
+### Minor
+
+- https://github.com/cds-snc/gcds-components/pull/306 - [c972bef](https://github.com/cds-snc/gcds-components/commit/c972befacb6961fed73c983df1fb4fda6cbc9ca5) - Add `gcds-heading` to component library
+
```
Comment (ethanWallace) [inline CHANGELOG.md:17]: Can you add https://github.com/cds-snc/gcds-components/pull/325 - [3ec8925](https://github.com/cds-snc/gcds-components/commit/3ec89251bf56b346df50ed67ad0a8992631240ff) - Fix slot CSS in `gcds-breadcrumbs-item` to display text underline as a patch to the changelog

**Review by ethanWallace — CHANGES_REQUESTED** [review]
Just one small thing in the changelog

File: CHANGELOG.md (hunk 1 above)
Comment (melaniebmn) [reply CHANGELOG.md:17]: Of course, sorry I missed that yesterday!

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Pull request merged**